pkg/assembler/graphql/schema/certifyVuln.graphql

```diff
@@ -129,7 +137,16 @@ VulnerabilityMetaDataInput represents the input for certifying vulnerability
 scans in mutations.
 """
 input VulnerabilityMetaDataInput {
```

Comment:
this would be used to create a new `VulnerabilityMetaData` so this is where we would specify the defaults if not specified by the user.  This would move into the new `VulnerabilityMetaData` graphQL definition file but here you would need to add in the defaults:
```
type VulnerabilityMetadataInputSpec {
...
cvss2Score: Float = 0.0
cvss3Score: Float = 0.0
links: [String!] = []
additionalInfo: String = ""
...
```